### PR TITLE
Fix DFU size attribute check

### PR DESCRIPTION
**Pull request by jameshilliard**
Fixes:
```python
    if args.size:
       ^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'size'
```

### CODE REVIEW

File: src/snagflash/dfu.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def dfu_cli(args):
 	dev.default_timeout = int(args.timeout)
 	for dfu_config in args.dfu_config:
 		(altsetting,sep,path) = dfu_config.partition(":")
-		if args.size:
```
Comment (rgantois) [inline src/snagflash/dfu.py:37]: This is actually a deprecated command line arg which is why you got an AttributeError. Could you please remove this check entirely?

File: src/snagflash/dfu.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def dfu_cli(args):
...
+		if hasattr(args, "size"):
```
Comment (rgantois) [inline src/snagflash/dfu.py:37]: This entire if/else block and the `if size is None` check a few lines below will be unecessary. Could you please remove those?

Comment (rgantois) [reply src/snagflash/dfu.py:37]: If you could also remove the relevant doc section (lines 37 to 41 in doc/snagflash.md) that would also be great.

**Review by rgantois — CHANGES_REQUESTED** [review]

File: src/snagflash/dfu.py (hunk 1 above)
Comment (jameshilliard) [reply src/snagflash/dfu.py:37]: removed

File: src/snagflash/dfu.py (hunk 2 above)
Comment (jameshilliard) [reply src/snagflash/dfu.py]: removed from docs as well

**Pull request merged**